anime-list-master.xml

```diff
@@ -33514,7 +33514,7 @@
   <anime anidbid="12244" tvdbid="321496" defaulttvdbseason="0" episodeoffset="" tmdbid="" imdbid="">
     <name>Koro-sensei Quest!</name>
   </anime>
-  <anime anidbid="12245" tvdbid="" defaulttvdbseason="" episodeoffset="" tmdbid="" imdbid="">
+  <anime anidbid="12245" tvdbid="283947" defaulttvdbseason="0" episodeoffset="13" tmdbid="" imdbid="">
```

Comment:
Since this is a movie, you can also add the tmdb and imdb IDs.

```suggestion
  <anime anidbid="12245" tvdbid="283947" defaulttvdbseason="0" episodeoffset="13" tmdbid="431808" imdbid="tt6475250">
```

Reply:
Sure, I don't usually check for these DBs since it's the flip of a coin whether an anime movie is on these and I don't really have an use for them, but will keep that in mind for the future :+1: